docs/source/notes/mps.rst

```diff
@@ -16,6 +16,12 @@ To get started, simply move your Tensor and Module to the ``mps`` device:
 
 .. code::
 
+    # Make sure the current PyTorch binary was built with MPS enabled
+    print(torch.backends.mps.is_built())
+    # And that the current hardware and MacOS version are sufficient to
+    # be able to use MPS
+    print(torch.backends.mps.is_available())
```

Comment:
can you have is_available but not is_built?  Might be better to reverse this, e.g. something like:

if !is_available:
   # print error message including whether it was built or not
 else run code